### PR TITLE
feat: Support env vars configuration for WebSocket server

### PR DESCRIPTION
**Pull request by benjreinhart**
### SUMMARY

The motivation for this PR is that application deployment typically involves referencing env vars to one degree or another. For example, some dynamic values in kubernetes (e.g., host ips) are easily exposed through env vars but not readily available otherwise. Additionally, it may make sense to have static configuration in the file and handle secrets (e.g., `jwtSecret`) a little differently.

### TEST PLAN

Unit tests, manual

### CODE REVIEW

File: superset-websocket/src/config.ts
Comment (villebro) [inline superset-websocket/src/config.ts:120]: not that it makes any difference, but I think the rest of the Superset codebase usually uses this notation:
```js
Object.entries(envVarConfigSetter).forEach(([envVar, set]) => { ... });
```

**Review by villebro — COMMENTED** [review]
LGTM, but waiting for @robdiciuccio to give formal approval

File: superset-websocket/spec/config.spec.ts
Comment (michael-s-molina) [inline superset-websocket/spec/config.spec.ts:21]: Thanks for this PR! I only have two nits.
- Can you rename it to `config.test.ts`? We're trying to make all tests adhere to the `.test `extension. 
- Can you inline the test? We're trying to [avoid unnecessary nesting](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing) to make the test results easier to read.

**Review by craig-rueda — APPROVED** [review]

File: superset-websocket/src/config.ts
Comment (benjreinhart) [reply superset-websocket/src/config.ts:120]: I didn't realize there was a preference in Superset already, happy to change in this in a subsequent PR!

Personally, I tend to favor the loops for things that mutate or have side effects, and the higher order functions for anything that has return values.

File: superset-websocket/spec/config.spec.ts
Comment (benjreinhart) [reply superset-websocket/spec/config.spec.ts:21]: Looks like this just got merged. I'm happy to make these changes. Will follow up with a PR

**Pull request merged**
